docs/code-of-conduct.rst

```diff
@@ -6,16 +6,30 @@ Code of Conduct
    :depth: 1
    :backlinks: none
 
-Write the Docs is a global community of documentarians who share information, discuss ideas, and work together to improve the art and science of documentation.
+Write the Docs is a global community of documentarians who share information, discuss ideas, and work together to improve the art and science of documentation. Everybody who participates in our community in one way or another is required to conform to this Code of Conduct (CoC). This includes online communications, meetup and conference attendees, speakers, sponsors, founders, moderators, organisers and volunteers.
 
-Diversity is one of our huge strengths, but it can also lead to communication issues. To support a welcoming environment for all, regardless of individual differences, we have a few ground rules that we ask people to adhere to when they participate in this community activities. These rules apply equally to founders, organizers, moderators, sponsors, and affiliates -- in short, to all participants.
+Why do we have a Code of Conduct?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-This isn't an exhaustive list of things that you must do, or can't do. Rather, take it in the spirit in which it's intended. It's a guide to make it easier to enrich all of us and the technical communities in which we participate, and which we represent.
+Our goals with having this Code of Conduct are:
+
+* Helping everyone feel safe and included. Many documentarians are new to our community, and some may have had poor experiences in other communities. We want to set the expectation that harassment and other unpleasant behaviour are not acceptable. If people do have an unpleasant experience, they'll know that’s neither the norm nor acceptable to us as a community.
+* Helping to build trust that if an incident is reported, we will do a thorough investigation, and not blame the reporter. Even, for example, if the incident concerns someone in a position of power.
+* Informing everyone of the expected behaviour. We are a diverse community, and having a Code of Conduct makes the expectations of everybody’s behaviour explicit and transparent.
+* Having a framework for report handling. The Code of Conduct is the basis for dealing with a report, assessing whether the CoC was violated, and what action should be taken.
+
+The Code of Conduct isn't an exhaustive list of things that you must do, or can't do. Rather, take it in the spirit in which it's intended. It's a guide to make it easier to enrich all of us and the communities in which we participate, and which we represent.
+
+If you believe someone is violating the Code of Conduct, please report it. Contact details for each space are listed at the end of this page.
 
 The principles
```

Comment:
It looks like the text isn't consistent between this page and the meetup CoC page. I'm wondering if we can extract the main principles into the /include/ folder and reuse them in both places (easier maintenance)

Reply:
This is correct - I tried to keep the meetup CoC page a bit shorter, so it contains two fragments of this text.